recipes/cudatoolkit/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# Originally forked from https://github.com/AnacondaRecipes/cudatoolkit-feedstock
+# Distributed under the BSD-2-Clause license
+# Copyright (c) 2017, Continuum Analytics, Inc. All rights reserved.
+{% set version = "11.0.3" %}
+{% set version_patch = "450.51.06" %}  # [linux64]
+{% set version_patch = "450.51.06" %}  # [linux-ppc64le]
+{% set version_patch = "451.82" %}  # [win]
+{% set major_minor = ".".join(version.split(".")[:2]) %}
+
+package:
+   name: cudatoolkit
+   # match the package version to the libcudart.so version
+   version: {{ version }}
+
+source:
+  - path: ./build.py
+  - url: https://developer.download.nvidia.com/compute/cuda/{{ version }}/docs/sidebar/md5sum.txt
+    sha256: 07e636b0e6525457052ecc2b43e36ab0352451edabb116eaea777756fbc39408
+  - url: https://developer.download.nvidia.com/compute/cuda/{{ version }}/local_installers/cuda_{{ version }}_{{ version_patch }}_linux.run  # [linux64]
+    md5: 70af4cebe30549b9995fb9c57d538214  # [linux64]
+  - url: https://developer.download.nvidia.com/compute/cuda/{{ version }}/local_installers/cuda_{{ version }}_{{ version_patch }}_win10.exe  # [win]
+    md5: 80ae0fdbe04759123f3cab81f2aadabd  # [win]
+
+build:
+  number: 0
+  script_env:
+    - NVTOOLSEXT_INSTALL_PATH
+    - DEBUG_INSTALLER_PATH
+  missing_dso_whitelist:
+    - "$RPATH/libdl.so.2"
+    - "$RPATH/libpthread.so.0"
+    - "$RPATH/libc.so.6"
+    - "$RPATH/librt.so.1"
+    - "$RPATH/ld-linux-x86-64.so.2"
+    - "$RPATH/libm.so.6"
+  skip: true  # [osx]
+  script: |
+    python build.py --version {{ version }} --version-patch {{ version_patch }}
+    if errorlevel 1 exit 1                                                 # [win]
+
+    :: copy nvvm and libdevice into the DLLs folder so numba can use them  # [win]
+    mkdir "%PREFIX%\DLLs"                                                  # [win]
+    xcopy /s /y "%PREFIX%\Library\bin\nvvm*" "%PREFIX%\DLLs\"              # [win]
+    if errorlevel 1 exit 1                                                 # [win]
+    xcopy /s /y "%PREFIX%\Library\bin\libdevice*" "%PREFIX%\DLLs\"         # [win]
+    if errorlevel 1 exit 1                                                 # [win]
+
+
+requirements:
+  build:
+    - 7za  # [win]
+    - python >=3.7
+    # for run_exports
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64 2.17  # [linux64]
+  run_constrained:
+    - __cuda >={{ major_minor }}
+
+test:
+  requires:
+    - numba
+    - python >=3.7
+    - setuptools  # for pkg_resources
+
+about:
+  home: https://developer.nvidia.com/cuda-toolkit
+  license: LicenseRef-NVIDIA-End-User-License-Agreement
```

Comment:
Conda-forge uses SPDX identifiers, so this is the equivalent identifier.  Is this an issue?

Reply:
If the text doesn't end up being `NVIDIA End User License Agreement` and link to https://docs.nvidia.com/cuda/eula/index.html then it will be an issue.

Reply:
OK, we'll just have to ignore the linter warnings then

Reply:
@kkraus14, can you please explain the `and link to https://docs.nvidia.com/cuda/eula/index.html` part? 

* conda-build does not allow the `license_file` entry to be a URL (it has to be a file)
* The artifacts at https://anaconda.org/nvidia/cudatoolkit/files have `license_file: NVIDIA_EULA` entries, as in the recipe here.
* Happy to include the PDF as this entry (available at https://docs.nvidia.com/cuda/pdf/EULA.pdf) instead, if needed

Happy to do whatever is needed.

Reply:
In the conda web UI there's an option to change the URL, i.e. at https://anaconda.org/nvidia/cudatoolkit/settings/options one can set the License URL if they have the proper permissions.

This was a sticking point in getting permission unfortunately.

Reply:
Ahh, I think that will have to happen once the feedstock is made and the package is uploaded then.

Reply:
Before merging it would be good to confirm that someone has the permission / ability to do this accordingly.

Reply:
Here is the current list of conda-forge owners on anaconda.org:

* beckermr
* bgruening
* cjwright4242
* dougal
* ericdill
* fwaters
* isuruf
* jakirkham
* jjhelmus
* marcelotrevisani
* mbargull
* msarahan
* mvn
* mwcraig
* ocefpaf
* patricksnape
* pelson
* scastellarin
* scopatz
* sodre
* SylvainCorlay
* uwe.korn
* wolfv

So at lease @jakirkham and I are on there